server/utils/nunjucksSetup.ts

```diff
@@ -21,6 +22,7 @@ export default function nunjucksSetup(app: express.Express): void {
   app.use((req, res, next) => {
     const externalUser = () => req.url.startsWith('/link') || req.url.startsWith('/barcode')
     app.locals.applicationName = externalUser() ? 'Send legal mail to prisons' : 'Check Rule 39 mail'
+    res.locals.gtmContainerId = externalUser() ? config.slmContainerId : config.checkRule39ContainerId
```

Comment:
Set the local `gtmContainerId` to either  `config.slmContainerId` or `config.checkRule39ContainerId`  (or null if the config values are not set)

The rest of the code to use the `gtmContainerId` in the Google Analytics javascript is already in place in `layout.njk` and has been there since I originally did the dev for this